getting-started/pull-request-lifecycle.rst

```diff
@@ -243,8 +242,8 @@ should do to help ensure that your pull request is accepted.
 ``patchcheck``
 ==============
 
-``patchcheck`` is a simple automated patch checklist that guides a developer
-through the common patch generation checks. To run ``patchcheck``:
+``patchcheck`` is a simple automated checklist for pull requests that guides a developer
```

Comment:
Normally you'd run `patchcheck` _before_ submitting the PR. Perhaps we could use the term "Git branch" here?

Reply:
I've never run patchcheck, I'm learning things :)

Reply:
I changed it to "checklist for changes in progress."

Reply:
(We've been moving a lot of patchcheck's functionality into pre-commit (https://github.com/python/cpython/issues/109408), it'd be nice to complete that and delete it.)